partition_gpu/partition_gpu.go

```diff
@@ -45,6 +45,8 @@ var partitionSizeToProfileID = map[string]string{
 	"3g.40gb": "9",
 	"4g.40gb": "5",
 	"7g.80gb": "0",
+	//nvidia-h100-80gb
+	"1g.20gb": "15",
```

Comment:
The comment is mis-leading. h100 actually has all the same profiles as nvidia-a100-80gb + "1g.20gb", please reflect this.

Reply:
Done.